frontend/cypress/integration/notification.spec.ts

```diff
@@ -0,0 +1,212 @@
+import 'cypress-localstorage-commands';
+
+describe('report.spec.ts - Report Page', () => {
+  describe('Activities done by Staff', () => {
+    //login as staff
+    before(() => {
+      cy.resetTestDataAndLoginAsStaff();
+      cy.saveLocalStorage();
+    });
+
+    beforeEach(() => {
+      cy.restoreLocalStorage();
+    });
+
+    describe('Equipment Activities', () => {
+      it('should successfully add an equipment', () => {
+        cy.visit('/equipment');
+        cy.intercept('POST', '/api/v1/equipment').as('createEquipment');
+        cy.intercept('GET', '/api/v1/equipment').as('getAllEquipment');
+
+        cy.getBySel('add-equipment-btn').click();
+
+        cy.getBySel('name-input').type('RJ45');
+        cy.getBySel('type-select').select('Wire');
+        cy.getBySel('brand-input').type('N/A');
+        cy.getBySel('serial-input').type('GIOEMSKIQ');
+        cy.getBySel('notes-input').type('Notes').wait(2000);
```

Comment:
ideally, we don't use numbers for waiting because some browsers/internet are faster/slower than others and it's unpredictable pero unya nalang e solve kay nag dali ta XD

Reply:
ay gi tuyo diay ni jaq? HAHA e ignore ra diay ako comment